web/pgadmin/browser/server_groups/servers/databases/schemas/tables/columns/utils.py

```diff
@@ -477,10 +480,19 @@ def parse_column_variables(col_variables):
     return spcoptions
 
 
-def fetch_geometry_details(data):
-    if 'typname' in data and (data['typname'] in ('geometry', 'geography')):
-        match_obj = re.search(r'([a-zA-Z]+),(\d+)', data['cltype'])
-        if match_obj:
-            data['geometry'] = match_obj.group(1)
-            data['srid'] = match_obj.group(2)
-    return data
+def fetch_length_precision_details(fulltype, data):
+    """
+    This function will fetch length and precision details
+    from fulltype.
+
+    :param fulltype: Full type.
+    :param data: Data.
+    """
+
+    match_obj = None
+    regex = r'\((\d*)([a-zA-Z]*),?(\d*)\)'
+    if data['typname'] in ('geometry', 'geography'):
```

Comment:
I don't understand why this is added. It kills the purpose of separating out the params parsing part.